app/components/CustomHeader/CustomHeader.tsx

```diff
@@ -0,0 +1,54 @@
+import { View } from 'native-base';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
+import React, { FC } from 'react';
```

Comment:
FC import as a type. 
do tsconfig.json in add below key to throwing error when not define only-type import
"isolatedModules": true,
 "preserveValueImports": true